theia-extensions/viewer-prototype/package.json

```diff
@@ -44,5 +44,6 @@
       "frontend": "lib/browser/trace-viewer/trace-viewer-frontend-module",
       "backend": "lib/node/trace-server-backend-module"
     }
-  ]
+  ],
+  "traceServerDefaultArgs": "-vmargs -Dtraceserver.port=8080"
```

Comment:
Remove (see explanation below)

Reply:
> Remove (see explanation below)

Set default command line arguments to an empty string, and resolved merge conflicts